spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/mapping/MappingRuleInterpreterGroupExplode.scala

```diff
@@ -24,8 +24,8 @@ import za.co.absa.enceladus.dao.EnceladusDAO
 import za.co.absa.enceladus.model.conformanceRule.{ConformanceRule, MappingConformanceRule}
 import za.co.absa.enceladus.model.{Dataset => ConfDataset}
 import za.co.absa.enceladus.utils.error._
-import za.co.absa.enceladus.utils.transformations.ArrayTransformations.arrCol
 import za.co.absa.enceladus.utils.udf.UDFNames
+import za.co.absa.spark.commons.sql.functions.col_of_path
```

Comment:
```suggestion
import za.co.absa.spark.commons.sql.functions.{col_of_path => colOfPath}
```

Reply:
I don't know. It's a Spark function, and for some reason, they keep their function name with snake case (perhaps because of SQL standards). It's also why I name the function this way, not the Java/Scala style camel case.

Reply:
Sure, I understand -- it is reasonable its original naming and usage witing Spark SQL as-is. My point was that when used in Scala code, you can just use a local alias (convenience of Scala import).